notebooks/en/agent_rag.ipynb

```diff
@@ -0,0 +1,2800 @@
+{
```

Comment:
<p>You switch between I and we often, try to be consistent</p><p>also dependencies*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thank you, I changed the phrasing to only use "we"!